src/main/resources/messages_en.properties

```diff
@@ -1041,6 +1041,9 @@ file.upload.brapi.images.filename.required=imageFileName is required
 file.upload.brapi.images.observationunitdbid.required=ObservationUnitDbId is required
 file.upload.entity.invalid=Only one of the following parameters should be provided: observationUnitUUID, germplasmUUID, instanceId, or lotId should be provided.
 file.upload.detach.parameters.invalid=Only one of the following parameters should be provided: observationUnitUUID, germplasmUUID, instanceId, or lotId should be provided.
+file.upload.reference.null=The request is invalid because some external reference information is either null or empty.
+file.upload.reference.id.exceeded.length=The request is invalid because referenceID must not exceed length of 2000.
```

Comment:
@mderamos I think we are not using "The request is invalid because" in our messages in general. 
the response code is already informing that is invalid, I think we can remove that from both messages